src/style.css

```
@@ -1 +1,61 @@
 /* Styles go here */
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
```

don't reset your margins and paddings with * selector
```suggestion
  box-sizing: border-box;
```